lib/protocol/src/openctx-protocol.schema.json

```diff
@@ -104,6 +104,18 @@
         }
       }
     },
+    "QuerySelector": {
+      "description": "List of regex patterns matching the query for which the provider can return context items.",
```

Comment:
Calling this query selector seems wrong based on the customer ask. If I am not mistaken it should be called message selector? IE they want to match the regex against the chat message submitted by the user?

Reply:
Okay yeah I checked ItemParams now. Thought the argument name was query.